bugbug/model.py

```diff
@@ -4,6 +4,7 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 import logging
+import os
```

Comment:
Importing the path module could be cleaner:
```suggestion
from os import path
```